### PR TITLE
Genome queries with EN files

### PR DESCRIPTION
**Pull request by daniel-ohayon**
- Added `GenomeQuery` class
- Added `FilesUtil.get_differential_expression_stats` method
- Added tests

Requires https://github.com/genestack/genestack-bio-applications/pull/841 to be merged first.

### CODE REVIEW

File: genestack_client/files_util.py [hunk 1]
```diff
@@ -622,3 +623,40 @@ def collect_metainfos(self, accessions):
...
+            - adjustedPValue
+            - genomeFeature (dict):
+
+                - featureType
+                - featureId
+                - subregions (list)
+                - featureName
+                - location (dict):
+
+                    - to
```
Comment (genestack-solomatin) [inline genestack_client/files_util.py]: 1 or 0 based?

File: genestack_client/files_util.py [hunk 2]
```diff
@@ -622,3 +623,40 @@ def collect_metainfos(self, accessions):
         :rtype: list[Metainfo]
         """
         return map(Metainfo.parse_metainfo_from_dict, self.invoke('getMetainfo', accessions))
+
+    def get_differential_expression_stats(self, accessions_to_queries):
+        """
+        Get differential expression statistics from files.
+        This method returns a dictionary where values are list of statistics dictionaries.
+        Each statistics dictionary has the following structure:
```
Comment (genestack-solomatin) [inline genestack_client/files_util.py]: Does all of this field always present?  What value shows empty field?

File: test/test_differential_expression.py [hunk 3]
```diff
@@ -85,5 +85,15 @@ def test_en_microarrays(conn, keep_files):
...
 
+def test_get_en_stats(conn):
+    fu = FilesUtil(conn)
+    query = GenomeQuery().set_order_ascending(True).set_limit(15).set_offset(0)
+    query.set_sorting_order(GenomeQuery.SortingOrder.BY_FDR).set_contrasts(["r1", "r2"])
+    query.add_filter(GenomeQuery.Filter.MAX_FDR, 0.2)
+    query.add_filter(GenomeQuery.Filter.MIN_LOG_FOLD_CHANGE, 0.1)
+    query.add_filter(GenomeQuery.Filter.REGULATION, GenomeQuery.Regulation.UP)
+
+    assert fu.get_differential_expression_stats({EN_TUTORIAL_FILE: query})
```
Comment (genestack-solomatin) [inline test/test_differential_expression.py]: May be add more deep check?

File: genestack_client/files_util.py [hunk 4]
```diff
@@ -637,17 +637,17 @@ def get_differential_expression_stats(self, accessions_to_queries):
...
-                - subregions (list)
+                - subregions (list)  (``[]`` if unavailable)
                 - featureName
                 - location (dict):
 
-                    - to
-                    - from (0-based)
-                    - contigName
-                    - normalizedContigName
+                    - to (``-1`` if unavailable)
```
Comment (genestack-solomatin) [inline genestack_client/files_util.py]: Can this field be `unavailable`?

Comment (daniel-ohayon) [inline genestack_client/files_util.py]: yes in the case of microarrays, we don't have information about where the genes map onto the genome, so `from` and `to` are not there

**Pull request merged**